<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrMethodClass</class>
 <widget class="QDialog" name="CorrMethodClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Correlation Methods</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="dialog_Frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <item>
     <widget class="QLabel" name="selectMethod_label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Correlation Method:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="methods_groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="method3_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Correlation Coefficient</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="method2_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Sum of Squared Differences</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="method1_radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cross Correlation</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>method1_radioButton</tabstop>
  <tabstop>method2_radioButton</tabstop>
  <tabstop>method3_radioButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
